js/fileactions.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2014
```

Comment:
The year for the copyright header needs to be updated

Reply:
There are a few files where this need updating

Reply:
Done